superset-frontend/plugins/plugin-chart-echarts/src/Timeseries/transformers.ts

```diff
@@ -225,6 +227,9 @@ export function transformSeries(
   } else if (stack && isTrend) {
     stackId = getTimeCompareStackId(forecastSeries.type, timeCompare, name);
   }
+  if (stackId && stackIdSuffix) {
+    stackId += stackIdSuffix;
```

Comment:
maybe a nit, but a cybersec person once said that you should always use `\n` as the separator in concatenated keys in cases where it's unlikely to appear naturally in the original keys. I'm not sure there's lots of collision risk here, but just in case I'd probably use `\n` instead of `_`.

Reply:
done 🙂